src/main/resources/application-dev.yml

```diff
@@ -0,0 +1,5 @@
+spring:
+  flyway:
+    # This should be removed when we think the DB is stable, and we should be moving onto migrations for changes.
```

Comment:
@timharrison-moj 

I have done this so we can make tweaks to the baseline SQL until it we think it is fairly settled.

This avoids the need to write extra migrations in the early stages of the project.

We can always take it out if it becomes a problem.  The trade off here being we will loose any dev data whilst this is like this.

Reply:
What is the effect of these?

Reply:
Without this we will have add (new) migrations for any changes to the base line DB.

It will mean we can loose what is in dev whilst this remains.  It would be nice not to have to add new migrations this early in development especially as things are likely to change.

The other alternative is drop the dev db when we make changes to the base line prior to deployment.

Reply:
But with it - what happens to the existing table when it is changed? Does it drop/recreate?